applications/plugins/SofaPython/Binding_Data.cpp

```diff
@@ -110,31 +112,34 @@ PyObject *GetDataValuePython(BaseData* data)
     const AbstractTypeInfo *typeinfo = data->getValueTypeInfo();
     const void* valueVoidPtr = data->getValueVoidPtr();
 
-    if (!typeinfo->Container())
+    if (!typeinfo->Container() || typeinfo->ValidInfo())
```

Comment:
What is the idea with typeinfo->ValidInfo()? Can you explain please?!

What does happen if the typeinfo is both a container and validinfo?
Is the idea is to remove the default implementation (returns a string) if !typeinfo->ValidInfo()?